router.go

```diff
@@ -79,19 +109,8 @@ func (r *Router) subpath(_path string, handlers []RequestHandler) (string, []Req
 
 func (r *Router) combineAndWrapHandlers(path, method string, handlers []RequestHandler) {
 	resultingPath, combinedHandlers := r.subpath(path, handlers)
-
 	fn := r.transformRequestHandlers(resultingPath, method, combinedHandlers)
-
-	switch method {
-	case http.MethodGet:
-		r.engine.actualRouter.GET(resultingPath, fn)
-	case http.MethodPost:
-		r.engine.actualRouter.POST(resultingPath, fn)
-	case http.MethodPut:
-		r.engine.actualRouter.PUT(resultingPath, fn)
-	case http.MethodHead:
-		r.engine.actualRouter.HEAD(resultingPath, fn)
-	}
+	r.engine.actualRouter.Handle(method, path, fn)
```

Comment:
The path should be actually resultingPath, because it is also used in groups, we need the total path